Classes/YLRefreshHeaderView/YLRefreshHeaderView.m

```diff
@@ -61,7 +61,7 @@ - (CGFloat)pullAmountToRefresh {
 
 #pragma mark UIScrollViewDelegate
 
-- (void)containingScrollViewDidScroll:(UIScrollView *)scrollView {
+- (void)containingScrollViewDidScroll:(nonnull UIScrollView *)scrollView {
```

Comment:
The main reason is that Xcode's autocomplete adds the annotations, but I think I like the extra documentation too.